AsyncDisplayKit/ASVideoNode.h

```diff
@@ -0,0 +1,34 @@
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+typedef NS_ENUM(NSUInteger, ASVideoGravity) {
+  ASVideoGravityResizeAspect,
+  ASVideoGravityResizeAspectFill,
+  ASVideoGravityResize
+};
+
+@protocol ASVideoNodeDelegate;
+
+@interface ASVideoNode : ASControlNode<_ASDisplayLayerDelegate>
```

Comment:
Important to not expose _ASDisplayLayerDelegate in the header (also have a space before the <).  Can you put a second @interface ASVideoNode () <_ASDisplayLayerDelegate> in the .m file?  Is it absolutely necessary to have that delegate conformation at all?  E.g. does ASControlNode or ASDisplayNode not already have that?